### ISSUE TITLE
Updating lwc-services from 3.0.4 to 3.1.0 is breaking ESLint

### ISSUE THREAD

**Issue by adityanaag3** [muenzpraeger/create-lwc-app#883]
### Summary

The lint script from package.json fails with the message `/bin/sh: eslint: command not found` after updating the version of lwc-services to 3.1.0. I tried manually installing eslint, but it keeps requesting for more and more packages to be installed.

### Steps To Reproduce:

1.  Update the version of lwc-services in package.json to  "lwc-services": "^3.1.0",
1.  Run `yarn install`
1.  Run `yarn lint`

### Expected result

Linting should've been successful.

### Actual result

`/bin/sh: eslint: command not found`  is thrown.

**Comment by muenzpraeger** [comment]
I recommend to check this with the LWC team. With version `3.1.1` of `lwc-services` I see this error:

```
It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:

    npm install eslint-plugin-import@latest --save-dev

The plugin "eslint-plugin-import" was referenced from the config file in ".eslintrc.json » @salesforce/eslint-config-lwc/recommended".
```

As that comes from the core LWC ESlint package it's out-of-scope for here. It is noteworthy that this does not apply to new projects that are created with `create-lwc-app`, so I'm unclear why this is happening.

FYI @pmdartus

**Comment by pmdartus** [comment]
With `@salesforce/eslint-config-lwc@1` all plugins this package depends upon have been moved to peer dependencies. Those missing dependencies should be added to the `lwc-services` package.json. 

The list of packages to install can be found in the [README](https://github.com/salesforce/eslint-config-lwc). More details about those changes can be found in the [release notes](https://github.com/salesforce/eslint-config-lwc/releases).

**Comment by muenzpraeger** [comment]
Crazy that a fresh install is not affected...

**Comment by muenzpraeger** [comment]
Fixed with #889
